### ISSUE TITLE
#8572 [mB] project overview: search, navigation and infobox on map (Philipp): Accessibility - Search bar is behind map pins in tab order

### ISSUE THREAD

**Issue by sblindow** [liqd/a4-meinberlin#5948]
**URL:** 
https://meinberlin-design-dev.liqd.net/
**expected behaviour:** 
Users should be able to comfortably navigate the map using the tab button. 
**behaviour:** 
The Search bar is behind the map-pins in the tab order, meaning that users have to go through all the pins before they can focus on the search bar and type something.

**Comment by vellip** [comment]
This is just how it is handled by leaflet. No real possibility to change this. See https://github.com/Leaflet/Leaflet/issues/7479

**Comment by CarolingerSeilchenspringer** [comment]
@vellip then we have to leave it that way.  thanks for looking into it.
